truetype/truetype_test.go

```diff
@@ -198,6 +198,48 @@ func TestIndex(t *testing.T) {
 	}
 }
 
+func TestName(t *testing.T) {
+	testCases := map[string]string{
+		"luxisr": "Luxi Sans",
```

Comment:
You might as well test luximr and luxirr as well. The optional x-foo-bar cases would be nice too, if you can get them.